contractsV5/bridges/bancor/KyberBancorReserve.sol

```diff
@@ -15,35 +15,40 @@ contract KyberBancorReserve is IKyberReserve, Withdrawable, Utils {
     bool public tradeEnabled;
     uint public feeBps;
 
-    IBancorNetwork public bancorNetwork; // 0x0e936B11c2e7b601055e58c7E32417187aF4de4a
+    IBancorNetwork public bancorNetwork;
 
-    IERC20 public bancorEth; // 0xc0829421C1d260BD3cB3E0F06cfE2D52db2cE315
-    IERC20 public bancorToken; // 0x1F573D6Fb3F13d689FF844B4cE37794d79a7FF1C
+    // Mainnet: Values for deploying and setting
```

Comment:
I assume those values should be verified.
maybe add a comment how to verify values.
for someone that is not you.
or even better, create a PR with deploy script for bancor including current values.

Reply:
Sure, will remove them from source code